tox.ini

```diff
@@ -9,40 +9,40 @@
 # something sensible.
 
 [tox]
-envlist = py{27,3}-test-coverage
+envlist = py{27,3}
 
-[testenv:py27-nose-coverage]
-commands = nose2 --with-coverage --coverage-report=annotate --coverage-report=term --coverage=nmoscommon
+[testenv:py27]
+commands =
+    coverage run --source=./nmoscommon -m unittest discover -s tests
+    coverage annotate
+    coverage report
 deps =
-    nose2
     coverage
     mock
 install_command = pip install --process-dependency-links {opts} {packages}
 
-[testenv:py3-nose-coverage]
-commands = nose2 --with-coverage --coverage-report=annotate --coverage-report=term --coverage=nmoscommon
+[testenv:py3]
+commands =
+    coverage run --source=./nmoscommon -m unittest discover -s tests
+    coverage annotate
+    coverage report
 deps =
-    nose2
     coverage
     mock
 install_command = pip install --process-dependency-links {opts} {packages}
 
-[testenv:py27-test-coverage]
-commands =
-    coverage run --source=./nmoscommon -m unittest discover -s tests
-    coverage annotate
-    coverage report
+[testenv:py27-nose-coverage]
```

Comment:
We should talk about this in Ways of Working - we probably don't need 4 permutations of running tests...

Reply:
Agreed. Whilst people may run multiple out of curiosity, it probably just complicates things in the general case, giving us more scaffolding to keep up to date.